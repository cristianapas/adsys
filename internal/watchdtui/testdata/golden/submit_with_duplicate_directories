Service adwatchd was successfully installed and is now running.
