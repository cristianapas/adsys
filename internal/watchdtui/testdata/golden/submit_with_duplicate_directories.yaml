verbose: 0
dirs:
- #ABSPATH#/foo
- #ABSPATH#/foo/bar
- #ABSPATH#/foo/baz
